models/_sector/nft/trades/chains/solana/platforms/_schema.yml

```diff
@@ -159,6 +159,82 @@ models:
        tests:
          - not_null
 
+  - name: magiceden_mmm_solana_trades
+    meta:
+      blockchain: solana
+      project: tensor
+      contributors: [ tserkityam ]
+    config:
+      tags: [ 'magiceden','solana','nft' ]
+    description: >
+      magiceden solana trades
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - project
+            - trade_category
+            - outer_instruction_index
+            - inner_instruction_index
+            - account_mint
+            - tx_id
+      - check_seed:
+          seed_file: ref('magiceden_mmm_solana_trades_seed')
+          match_columns:
+            - block_slot
+            - tx_id
+            - outer_instruction_index
+            - inner_instruction_index
+          check_columns:
+            - buyer
+            - seller
+            - account_mint
+            - amount_raw
+            - taker_fee_amount_raw
```

Comment:
The current dbt failure may be caused by the `double` typed comparison:
https://dune.com/queries/3440103

You can remove those double typed columns in `check_columns`.

Reply:
Thanks @Hosuke , I removed taker and maker fee from the checked column list . PTAL.